src/cc_catalog_airflow/dags/provider_api_scripts/smithsonian.py

```diff
@@ -473,7 +473,6 @@ def _process_image_list(
 if __name__ == '__main__':
     logging.basicConfig(
         format='{asctime} - {name} - {levelname}:  {message}',
-        style='{',
```

Comment:
I guess we'd unify the logging format here to: `%(some_variable)` as in #100. Aside from that, is looking good 👍

Reply:
Thank you, I've missed this!